CreeDictionary/tests/DatabaseManager_tests/xml_importer_test.py

```diff
@@ -57,19 +57,6 @@ def test_import_xml_common_analysis_definition_merge(shared_datadir):
     assert Wordform.objects.get(text="pisiniw").definitions.count() == 2
 
 
-@pytest.mark.django_db
-def test_import_xml_crkeng_small_duplicate_l_pos_ic(shared_datadir):
-    # This test shows the behavior of the importer when entries with duplicate (l, pos, ic) in the xml file exists
-    # It's a rare case
-
-    # two Wordform objects will be created, each with the pooled translations from the two entries
-    migrate_and_import(shared_datadir / "crkeng-small-duplicate-l-pos-ic")
-
-    lemmas = Wordform.objects.filter(text="asawâpiwin", is_lemma=True)
```

Comment:
@aarppe this test is a left over from old days where the dictionary content would have duplicate entries with _sometimes_ different definitions. This caused some entries to have duplicates here. In addition `is_lemma` used to refer to _all dictionary heads_ — not lemmas, due to a misunderstanding in terminology 😬